src/components/organism/PoolLiquidity/index.tsx

```diff
@@ -311,15 +311,17 @@ const PoolLiquidity = () => {
         title={t("button.selectToken")}
       />
 
-      <PoolAndLiquidityCreateSuccessModal
-        // setIsModalOpen={setIsSuccessModalOpen}
+      <SwapAndPoolSuccessModal
         open={isSuccessModalOpen}
         onClose={closeSuccessModal}
         contentTitle={poolExists ? "Successful Added Liquidity" : "Pool Successfully Created"}
-        nativeTokenAmount={selectedTokenNativeValue.tokenValue}
-        assetTokenAmount={selectedTokenAssetValue.tokenValue}
-        nativeTokenSymbol={selectedTokenA.nativeTokenSymbol}
-        assetTokenSymbol={selectedTokenB.tokenSymbol}
+        tokenBValue={selectedTokenNativeValue.tokenValue}
+        tokenAValue={selectedTokenAssetValue.tokenValue}
+        tokenBSymbol={selectedTokenA.nativeTokenSymbol}
+        tokenASymbol={selectedTokenB.tokenSymbol}
+        tokenAIcon={<DotToken />}
+        tokenBIcon={<DotToken />}
+        actionLabel="added"
```

Comment:
Why don't u change this to one property instead of 6 properties? Eg: tokens: {tokenA: {value:'1234', symbol: 'test'}, {tokenB: {value: '123', symbol: 'testt', icon: <DotToken />}}} it is much cleaner